packages/sampler/stories/default/VirtualList.js

```diff
@@ -45,23 +44,23 @@ const updateDataSize = (dataSize) => {
 
 updateDataSize(defaultDataSize);
 
-const UiVirtualListConfig = mergeComponentMetadata('VirtualList', UiVirtualListBase, UiScrollableBase);
+const VirtualListConfig = mergeComponentMetadata('VirtualList', VirtualList, VirtualListBasic);
```

Comment:
The default props for the `VirtualListBasic` overrides the default props for the `VirtualList`.
Currently, all the default props are same between them. As the aspect of the maintenance point, it would be better to switch them.

Actually, in theme samplers, there might be problems if defining the `VirtualList` first.

The `VirtualListBasic` should be in front of the `VirtualList`.